tests/qnn/test_cost.py

```diff
@@ -21,6 +21,9 @@
 from pennylane.qnn.cost import SquaredErrorLoss
 
 
+ALLOWED_INTERFACES = ["tf", "jax", "autograd", "torch"]
```

Comment:
Hmm, it seems that this list will have to be maintained, how come it had to be redefined?

Reply:
I am not a fan of parametrizing tests over different logical pathways. Test parametrization should only be used for _data_ that is input to the same logic.

Further, I also do not think it is a good idea to parametrize a test based on data from within the source code itself - this can lead to

* confusion when maintaining the tests
* issues where changes to unrelated parts of the code base may break this test.

Better to be explicit in the test, and remember to update this test when adding a new interface.